2018-October-30/writingSamples/heslin.tex

```diff
@@ -0,0 +1,170 @@
+memory, attention, and decision making. Importantly, interval timing
+task measures can be readily compared across animal models,
+neuropsychiatric patients, and healthy human comparisons. Therefore, it
+is an ideal type of task with which to probe our circuit and neuronal
+population of interest in rodent models. Specifically, we will
+investigate how LCN D1DR-expressing neurons contribute to performance in
+a peak interval timing task, and how performance correlates with
+population activity in the LCN and prelimbic cortex (PL), the rodent
+medial frontal cortex. Based on our prior work and the primary
+literature, our \textbf{\emph{overall hypothesis}} is that inhibitory
+D1DR-expressing cells of the LCN are necessary for successful interval
+timing through modulation of activity in the PL.
+
+\textbf{\emph{Aim 1. Determine if LCN D1DR-expressing neurons are
+necessary for interval timing}}
+
+The cerebellum and LCN specifically have been implicated in both
+sub-second and supra-second interval timing\textsuperscript{23,27--30}.
+Dopaminergic signaling is crucial for timing behavior and is one of
+several abnormal neurotransmitter systems in
+schizophrenia\textsuperscript{4,21}. The role of inhibitory
+D1DR-expressing neurons in the LCN is presently unknown. We will
+determine if LCN D1DR-expressing neurons are necessary for interval
+timing performance using optogenetically driven inactivation during the
+task. Additionally, synchronization of cortical pyramidal cell activity
+by inhibitory interneurons is associated with cognitive
+processing\textsuperscript{31,32}. Therefore, we will record local field
+potentials (LFPs) and single neurons during optogenetic inactivation to
+assess whether LCN D1DR-expressing neurons contribute to synchronized
+activity in the LCN and PL. Combining optogenetic and
+electrophysiological methods will allow us to test the \emph{hypothesis}
+that inactivation of LCN D1DR-expressing neurons results in interval
+timing deficits and decreased synchronized neuronal activity in the LCN
+and PL.
+
+\textbf{\emph{Aim 2. Determine if stimulating LCN D1DR-expressing
+neurons compensates for PL dysfunction }}
+
+Cerebellar stimulation may reduce cognitive symptoms in
+schizophrenia\textsuperscript{33} but it is unknown to what extent and
+by which mechanisms. We can pharmacologically mimic frontal cortex
+dysfunction, allowing us to determine whether optogenetic stimulation of
+LCN D1DR-expressing neurons rescues interval timing performance and
+impacts neuronal activity in the PL. This design will allow us to test
+the \emph{hypothesis} that stimulation of LCN D1DR-expressing neurons
+can compensate for aberrant activity in the PL and improve interval
+timing performance. Here we will measure `compensation' based on
+improvement in interval timing task performance precision and increased
+frontal cortex single unit synchronization and LFP 1-4Hz power.
```

Comment:
could add another impact statement here and how it relates to your over-arching hypothesis.